### ISSUE TITLE
free(): invalid pointer: 0x.... with triple-quotes in description field

### ISSUE THREAD

**Issue by paoloschi** [johanmalm/jgmenu#41]
this has happened to me since you have implemented the triple-quotes but before reporting it I waited to be able to try by running __jgmenu v0.7__ from the official Void package (not trusting a lot in my own _builds_)..

if I take the example from the manual and launch it in this way

```
echo "foo,\"\"\"^pipe(find /usr/share/pixmaps -printf '%f,display %p,%p\n')\"\"\"" | jgmenu --simple
info: got font from gtk config file
info: set font to 'Fira Sans 10'
info: got icon theme from gtk config file
info: set icon theme to 'gnome-wine'
info: cleaning up...
```

everything works as it should!
now I'll add a comma in the _description_ field also which, in the same way, I protect with triple-quotas:

```
echo "\"\"\"foo, bar\"\"\",\"\"\"^pipe(find /usr/share/pixmaps -printf '%f,display %p,%p\n')\"\"\"" | jgmenu --simple
info: got font from gtk config file
info: set font to 'Fira Sans 10'
info: got icon theme from gtk config file
info: set icon theme to 'gnome-wine'
info: cleaning up...
*** Error in `jgmenu': free(): invalid pointer: 0x0000556477b822c3 ***
```

The menu again worked and the description field correctly showed the comma included in the label.
But this time jgmenu exits with __error code 134__

the same by simplifying the command:
```
echo "\"\"\"foo, bar\"\"\",true" | jgmenu --simple
info: got font from gtk config file
info: set font to 'Fira Sans 10'
info: got icon theme from gtk config file
info: set icon theme to 'gnome-wine'
info: cleaning up...
*** Error in `jgmenu': free(): invalid pointer: 0x000056461bb50523 ***
```

In practice, systematically I have the error `free(): invalid pointer` every time that I need to introduce triple-quotes __but only in the _description_ field__!
same thing if I use `printf` instead of `echo` (or even `/usr/bin/echo` instead of the internal bash echo command) or if I put the same definition in a CSV file

it is not clear to me if I am wrong in executing the command, if it is something related to the Void distribution or if it may be a bug of jgmenu

**Comment by johanmalm** [comment]
Oh dear. I understand what's happening. Thanks for spotting it and letting me know.
It's a bug in argv-buf.c
I'm onto it.

**Comment by johanmalm** [comment]
I think I've fixed it. Thanks for the comprehensive notes. They made it easy for me to the spot the problem.

I've released v0.7.1 to include this fix.
